website/content/docs/getting-started/create.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Create a Kind cluster"
```

Comment:
I think it would be better to do with this a single node k0s but I wasn't sure on the status of this. Our current k0s setup involves AWS and TF. I used kind to keep it simple for now.